nox/command.py

```diff
@@ -84,7 +90,7 @@ def run(
         success_codes = [0]
 
     cmd, args = args[0], args[1:]
-    full_cmd = f"{cmd} {' '.join(args)}"
+    full_cmd = f"{cmd} {_shlex_join(args)}"
```

Comment:
It occurs to me I should probably do a docs search for `python -m pip` and see what should be updated.

(As reviewers may already see from the branch name, I made this PR fully from the browser.)